utils/revive-builder-debian.dockerfile

```diff
@@ -0,0 +1,14 @@
+# syntax=docker/dockerfile:1
+# Dockerfile for building revive in a Debian container.
+FROM debian:12
+RUN <<EOF
+apt-get update
+apt-get install -q -y build-essential cmake make ninja-build python3 \
+    libmpfr-dev libgmp-dev libmpc-dev ncurses-dev \
+    git curl
+EOF
+ARG RUST_VERSION=stable
```

Comment:
Same here, I think sticking to specific version should be better.

What is missing is a rust version in the top level `Cargo.toml`:
```
rust-version = "1.80.0"
```

Could you add this here and to the top level Cargo.toml?

Reply:
The value in the Dockerfile is a default value, the value used is set in in the workflow file.

Changed the Cargo.toml file to specify the rust-version is 1.80